packages/medusa/src/services/cart.ts

```diff
@@ -87,6 +89,7 @@ class CartService extends TransactionBaseService<CartService> {
   protected readonly addressRepository_: typeof AddressRepository
   protected readonly paymentSessionRepository_: typeof PaymentSessionRepository
   protected readonly lineItemRepository_: typeof LineItemRepository
+  protected readonly salesChannelRepository_: typeof SalesChannelRepository
```

Comment:
suggestion: As we try to avoid injecting the repository, could you instead create a service method and inject the service instead of the repo here

Reply:
yes, just an unused dep, will be rm